noxfile.py

```
@@ -0,0 +1,325 @@
+    Any existing container with the same name is removed first to ensure
+    a clean database state. The container is destroyed after the test run
+    by ``_destroy_postgres``.
+    """
+    # Always remove any existing container to guarantee a clean database
+    if _container_exists(POSTGRES_CONTAINER_NAME):
+        session.log(f"Removing existing container '{POSTGRES_CONTAINER_NAME}'...")
+        subprocess.run(["docker", "rm", "-f", POSTGRES_CONTAINER_NAME], check=True)
+
+    # Build the image
+    session.log("Building PostgreSQL Docker image...")
+    subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)